### PR TITLE
JIRA: VZ-9070 remove additional k3s references

### PR DESCRIPTION
**Pull request by jmaron99**
Removing some additional k3s references and related code.  The only code remaining now is in the scripts directory and some test code modules. These can be removed as part of additional cleanup tasks associated with the phase 2 pruning.

### CODE REVIEW

File: pkg/api/norman/customization/cluster/validator.go [hunk 1]
```diff
@@ -805,3 +746,30 @@ func validateGKEPrivateClusterConfig(spec *v32.ClusterSpec) error {
 	}
 	return nil
 }
+
+// isNewerVersion returns true if updated versions semver is newer and false if its
+// semver is older. If semver is equal then metadata is alphanumerically compared.
+func isNewerVersion(prevVersion, updatedVersion string) (bool, error) {
```
Comment (bobdonat) [inline pkg/api/norman/customization/cluster/validator.go]: I don't think this function is used

Comment (mgianatagh) [inline pkg/api/norman/customization/cluster/validator.go]: This function does not appear to be used.  And appears to be defined again in pkg/image/external/external.go

File: rancher.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mgianatagh) [inline rancher.iml]: Is this new file required?

File: pkg/api/norman/customization/cluster/validator.go (hunk 1 above)
Comment (jmaron99) [reply pkg/api/norman/customization/cluster/validator.go]: Strange - I think it was used at one point.  I'll remove it.

Comment (jmaron99) [reply pkg/api/norman/customization/cluster/validator.go]: The latter is internal and is being used locally in that file.  I think the former stopped being used due to some additional code removal.

File: rancher.iml (hunk 2 above)
Comment (jmaron99) [reply rancher.iml]: No.  Will remove.

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**